app/controllers/application_controller.rb

```diff
@@ -22,9 +24,24 @@ class ApplicationController < ActionController::Base
      render_404 exception
   end
 
+  def profile
```

Comment:
How do we use this profiling tool on the app? Mind putting together a little doc on developers-handbook?